wikipedia-dark.user.css

```diff
@@ -40,12 +40,40 @@
   Fixed "Fixed" <<<EOT
   fixed EOT;
 }
+@advanced dropdown custom_scrollbar "Custom Scrollbar:" {
+		1. "*ON" <<<EOT ::-webkit-scrollbar-track {
+             -webkit-box-shadow: inset 0 0 0px rgba(0, 0, 0, 0.3);
+             border-radius: 0px;
+             background-color: #212121;
+        }
+        html[hide-scrollbar="true"] ::-webkit-scrollbar{display:none}
+        ::-webkit-scrollbar {
+             width: var(--custom_scrollbar_width);
+             background-color: #212121;
+        }
+        ::-webkit-scrollbar-thumb {
+             border-radius: 0px;
+             -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, .6);
+             background-color: var(--custom_scrollbar_color);
+        }
+		::-webkit-scrollbar-corner {
+			 background-color: #212121;
+			 border-color: #212121;
+		}EOT;
+	2. "OFF" <<<EOT EOT;
+}
+@advanced text custom_scrollbar_width "Width of custom scrollbar:(px, em, rem, vh etc.)" 1em
+@advanced text custom_scrollbar_color "Color of custom scrollbar:(hex, rgb, hsla or variables)" var(--assess-1)
+
+
 @preprocessor uso
 ==/UserStyle== */
 @-moz-document
-regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
+regexp("https?:\/\/.*wik(i|t).*(org|jp).*"), domain("sonichu.com"), domain("diff.wikimedia.org") {
   /* Global color palette */
   :root {
+	  --custom_scrollbar_width: /*[[custom_scrollbar_width]]*/;
+	  --custom_scrollbar_color: /*[[custom_scrollbar_color]]*/;
```

Comment:
no scrollbars in style, thanks, use overlay scrollbars style.

Reply:
yup I got it